client/src/screens/auth/components/SignIn.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, ActivityIndicator } from 'react-native';
+
+import { Container, Holder } from '../../../components/Container';
+import { Input } from '../../../components/Input';
+import { Button } from '../../../components/Button';
+
+class SignIn extends Component {
+  state = {
+    username: '',
+    password: '',
+  }
+
+  onChangeUsername = (value) => {
+    this.setState({
+      username: value,
+    });
+  };
+
+  onChangePassword = (value) => {
+    this.setState({
+      password: value,
+    });
+  };
+
+  onSignIn = () => {
+    const { username, password } = this.state;
+    this.props.onSignIn({ username, password });
+  }
+
+  render() {
+    const { loading } = this.props;
+    const { username, password } = this.state;
+    return (
+      <Container>
+        <Holder margin transparent>
+          <Input
+            type="username"
+            placeholder="Username"
+            value={username}
+            onChangeText={this.onChangeUsername}
+          />
+          <Input
+            type="password"
+            placeholder="Password"
+            value={password}
+            onChangeText={this.onChangePassword}
+          />
+          <Button text="Sign In" onPress={this.onSignIn} disabled={loading} />
+          {loading && (
+            <View
+              style={{
```

Comment:
We have a component to do what this view is doing. Think it's called Container Middle or something like that.

Reply:
yeah got a follow up PR for that